### PR TITLE
Add BobaBot (Fixes #43)

### LINKED ISSUE #43: Question 1 Page: Boba Bot
## Description
This component should be approached with reuseability in mind. We should keep in mind the text within the bubble will be changing, so the component should include a prop to allow for that

image

### PR DESCRIPTION
**Pull request by jshiii117**
#### What is the purpose of your change?
- Introducing the reusable BobaBot component

#### What areas of the application does this change impact? (changes API, folder structure, etc)
- New component

#### What features/fixes are deferred/not handled in this PR?
- 

## Checklist
- [ ] Adds unit tests
- [X] Has a work item linked to the PR
image
image

### CODE REVIEW

File: tmt-app/src/Components/BobaBot.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+          top: "10%",
+          width: "70%",
+        }}
+      >
+        <Typography
+          sx={{
+            fontWeight: "bold",
+            textAlign: "center",
+          }}
+        >
```
Comment (jwu910) [inline tmt-app/src/Components/BobaBot.jsx:31]: Does this need a variant?

**Comment by jwu910** [comment]
Are the screen shots above from your testing?

File: tmt-app/src/Components/BobaBot.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+function BobaBot({ text }) {
+  return (
+    <Box
+      sx={{
+        alignItems: "center",
+        display: "inline-flex",
+        position: "relative",
+      }}
+    >
+      <img src={BobaBotLarge} alt={"Mascot Icon"} />
```
Comment (jwu910) [inline tmt-app/src/Components/BobaBot.jsx:14]: Alt messages can and should be more descriptive. In this case I would add the prop text to the alt. Something like "mascot icon with speech bubble that says..."

File: tmt-app/src/Components/BobaBot.jsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      <Box
+        sx={{
+          alignItems: "center",
+          display: "flex",
+          height: "40%",
+          left: "15%",
+          position: "absolute",
+          top: "10%",
+          width: "70%",
```
Comment (jwu910) [inline tmt-app/src/Components/BobaBot.jsx:23]: Does this look okay on different view ports?

Comment (michachan) [reply tmt-app/src/Components/BobaBot.jsx:23]: I'm also concerned about the use of unclamped, static percentages here

**Review by michachan — APPROVED** [review]

File: tmt-app/src/Components/BobaBot.jsx (hunk 1 above)
Comment (jshiii117) [reply tmt-app/src/Components/BobaBot.jsx:31]: I didn't see any specifications for variants in the Specs for Devs file in Figma? Should defining the variants for text be something the devs do?

**Review by jwu910 — APPROVED** [review]
I think we can merge for now's

**Pull request merged**